integration/hive/src/main/scala/org/apache/carbondata/hiveexample/HiveExample.scala

```diff
@@ -49,17 +49,19 @@ object HiveExample {
 
     sparkSession.sql("""DROP TABLE IF EXISTS HIVE_CARBON_EXAMPLE""".stripMargin)
 
-    sparkSession
-      .sql(
-        "CREATE TABLE HIVE_CARBON_EXAMPLE (ID int,NAME string,SALARY double) " +
-          "STORED BY 'CARBONDATA' ")
+    sparkSession.sql(
+      s"""
+         | CREATE TABLE HIVE_CARBON_EXAMPLE
+         | (ID int,NAME string,SALARY double)
+         | STORED BY 'CARBONDATA'
```

Comment:
can you use lower case carbondata? like: STORED BY 'carbondata'

Reply:
```suggestion
         | STORED AS CARBONDATA
```

Reply:
@jackylk This suggestion doesn't work
@xubo245 I have already lower CARBONDATA